examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -66,55 +67,62 @@ export class TaskList extends DataObject implements ITaskList {
      * collection pattern -- see the contact-collection example for more details on this pattern.
      */
     private readonly tasks = new Map<string, Task>();
+    // savedData stores data retrieved from the external source.
```

Comment:
Nit: use the same comment format as the rest of the doc. :)
```suggestion
    /**
     * savedData stores data retrieved from the external source.
     */
```